start-splice-cluster

```diff
@@ -188,9 +188,9 @@ echo "Starting Master and 1 Region Server. Log file is ${SPLICE_LOG}"
 ## (master + region server on 1527)
 
 if [ -z "${SERVER_SSLMODE}" ]; then
-  (${MVN} exec:exec -P${PROFILE},spliceFast ${SYSTEM_PROPS} -Dxml.plan.debug.path=${DEBUG_PATH} > ${SPLICE_LOG} 2>&1) &
+  (${MVN} exec:exec -P${PROFILE},spliceFast,jacoco  ${SYSTEM_PROPS} -Dxml.plan.debug.path=${DEBUG_PATH} > ${SPLICE_LOG} 2>&1) &
```

Comment:
The 'jacoco' mode should not be used unconditionally in every standalone run.

Reply:
only when '-Djacoco' and -Pjacoco both provided ,the code coverage anlysis will be enabled ,so it won’t be used unconditionally in every standalone run, when '-Djacoco' is not provided, the jacocoeagent will be '-server',which is harmless I think

Reply:
How are you going to provide '-Djacoco' for start-splice-cluster? 
Why do you need '-Pjacoco' in start-splice-cluster at all?

Reply:
You are right,It is unnecessary  for code coverage I have removed it